### DISCUSSION TITLE
Q: 提交评论Failed to fetch || Q: Submit comment Failed to fetch

### DISCUSSION THREAD

**Discussion by jimes3** [walinejs/waline#2244] [Q&A]
### Describe the bug | 问题描述

waline服务网站是可以评论的，但是博客里面就不能了，我现在初步怀疑是没有html引入，但是我不知道怎么引入，文件里也没有什么html文件啊，求救求救！serverurl我都填的是那个可以运行的waline评论网站，应该没有问题。

### Website URL | 问题网站

https://jimes.cn/2024/01/06/hello-world/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

You can comment on the waline service website, but not on the blog. I now initially suspect that there is no html introduced, but I don’t know how to introduce it. There is no html file in the file. Please help! The serverurl I filled in is the waline review website that can be run, so there should be no problem.

### Website URL | Problem website

https://jimes.cn/2024/01/06/hello-world/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by jimes3** [comment]
博客有时候评论区会不见，刷新一下就出来了。

---
Sometimes the comment section of the blog will disappear, and it will appear after refreshing it.

**Reply by CarefreeMontage** [reply]
去Vercel你部署的waline项目里setting下的Environment Variables里面把SMTP_SECURE这一项删了应该就可以了，或者改成false

---
Go to the Environment Variables under the settings of the waline project you deployed in Vercel and delete the SMTP_SECURE item. It should be ok, or change it to false
